lib/archethic_web/controllers/api/origin_public_key_payload.ex

```diff
@@ -0,0 +1,49 @@
+defmodule ArchEthicWeb.API.OriginPublicKeyPayload do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias ArchEthic.Utils
+
+  alias ArchEthicWeb.API.Types.Hex
+  alias ArchEthicWeb.API.Types.PublicKey
+
+  embedded_schema do
+    field(:PublicKey, PublicKey)
```

Comment:
Please use camelCase standard to be consistent

Reply:
Done